Riot/Modules/Room/DataSources/RoomDataSource.m

```diff
@@ -1095,83 +1087,6 @@ - (void)updateStatusInfo
     }
 }
 
-#pragma mark - Room creation intro cell
-
-- (BOOL)canShowRoomCreationIntroCell
-{
-    NSString* userId = self.mxSession.myUser.userId;
-
-    if (!userId || !self.isLive || self.isPeeking)
-    {
-        return NO;
-    }
-    
-    // Room creation cell is only shown for the creator
-    return [self.room.summary.creatorUserId isEqualToString:userId];
-}
-
-- (void)enableRoomCreationIntroCellDisplayIfNeeded
-{
-    self.showRoomCreationCell = [self canShowRoomCreationIntroCell];
-}
-
-// Insert the room creation intro cell at the begining
-- (void)insertRoomCreationIntroCellDataIfNeeded
```

Comment:
None of this should be neccesssary because the "m.room.create" event is a regular part of the `bubbles` array, so will be displayed like any other once available from the pagination